src/filter.ts

```diff
@@ -110,8 +110,7 @@ export function expression(model: Model, filter: Filter): string {
   if (isString(filter)) {
     return filter;
   } else if (isSelectionFilter(filter)) {
-    const selection = model.getSelectionComponent(filter.selection);
-    return predicate(model, filter.selection, selection.type, selection.resolve, null);
+    return predicate(model, filter.selection);
```

Comment:
FYI, `selection` in SelectionFilter is still a string, not a `LogicalOperand` yet